lib/main.go

```diff
@@ -50,6 +50,25 @@ func Run() {
 
 	rand.Seed(time.Now().Unix())
 
+	//
+	// worker also initialize a calclent with the same poolname using threadid==0 in
```

Comment:
Why did we make this change? Pls revert.

Reply:
Done